data/json/monsters/feral_humans.json

```
@@ -95,7 +97,71 @@
     "melee_dice": 2,
     "melee_dice_sides": 7,
     "melee_cut": 9,
-    "death_drops": "feral_humans_death_drops_axe"
+    "death_drops": "feral_humans_death_drops_axe",
+    "fungalize_into": "mon_feral_human_axe_fungal_infected"
+  },
+  {
+    "id": "mon_feral_human_pipe_fungal_infected",
+    "type": "MONSTER",
+    "description": "Delicate fungal stalks sprout in rows from sores in the arms, and the facial expression is pure joy.  Coos softly.",
+    "copy-from": "mon_feral_human_pipe",
+    "default_faction": "fungus",
+    "speed": 0,
+    "upgrades": { "age_grow": 4, "into": "mon_feral_human_pipe_fungal_corpse" },
+    "burn_into": "mon_starer_pipe",
+    "extend": { "flags": [ "NO_FUNG_DMG" ] }
+  },
+  {
+    "id": "mon_feral_human_crowbar_fungal_infected",
+    "type": "MONSTER",
+    "description": "Delicate fungal stalks sprout in rows from sores in the arms, and the facial expression is pure joy.  Coos softly.",
+    "copy-from": "mon_feral_human_crowbar",
+    "default_faction": "fungus",
+    "speed": 0,
+    "upgrades": { "age_grow": 4, "into": "mon_feral_human_crowbar_fungal_corpse" },
+    "burn_into": "mon_starer_crowbar",
+    "extend": { "flags": [ "NO_FUNG_DMG" ] }
+  },
+  {
+    "id": "mon_feral_human_axe_fungal_infected",
+    "type": "MONSTER",
+    "copy-from": "mon_feral_human_axe",
+    "upgrades": { "age_grow": 4, "into": "mon_feral_human_axe_fungal_corpse" },
+    "burn_into": "mon_starer_axe",
+    "extend": { "flags": [ "NO_FUNG_DMG" ] }
+  },
+  {
+    "id": "mon_feral_human_pipe_fungal_corpse",
+    "type": "MONSTER",
+    "description": "Delicate fungal stalks sprout in rows from sores covering the body, and the facial expression is pure joy.  Coos softly.",
```

```suggestion
    "description": "Delicate fungal stalks sprout in rows from sores covering the body, and the facial expression is pure joy.  Coos softly.",
```
This is mostly what I'm looking for, but a fungal corpse probably doesn't have a joyful expression. This is the player after their arms have broken and they've watched the last few moments of their life. They're in intense pain.